src/Tests/Microsoft.NET.Sdk.BlazorWebAssembly.Tests/BlazorWasmStaticWebAssetsIntegrationTest.cs

```diff
@@ -90,7 +90,7 @@ public void StaticWebAssets_PublishMinimal_Works()
                 intermediateOutputPath);
         }
 
-        [Fact]
+        [Fact(Skip="https://github.com/dotnet/sdk/issues/28429")]
```

Comment:
You could use a `RequiredMSBuildVersionFact` instead so that the test will automatically start running again.

Reply:
do you know which MSBuild version the fix is in and can we do preview versions like that?